README.md

```diff
@@ -30,10 +30,12 @@ Is this project useful to you? Give me a [**⬆money upvote!⬆**](https://donat
 
 ### Using manual steps 
 - `npm install` 
-- Clone the submodules - `git submodule update --recursive`
+- Clone the submodules - `git submodule init && git submodule update --recursive`
 - Run `npm install` in `whisper.cpp/examples/addon.node`
 - Run `npm install` in `llama.cpp/examples/addon.node`
 - Build & run them (make sure that whisper.cpp & llama.cpp can run)
+  -  `cd whisper.cpp && make`
+  -  `cd llama.cpp && make`
```

Comment:
I would suggest also adding in steps to verify the models run (even if and probably better if it is a link to the README on where to do that, otherwise talk may see more issues from people who unsuccessfully installed either of these models) or atleast add something like

```
make -j && ./main -m ./models/7B/ggml-model-q4_0.bin -p "Building a website can be done in 10 simple steps:" -n 512
```
below llama make (also brings up whether or not -j should be added to decrease compile times and use more threads to instructions)

Reply:
Thanks, will commit this direct as well